processing/bloom_handler.go

```diff
@@ -131,17 +132,29 @@ func MakeBloomHandler(iocBloom *bloom.BloomFilter,
 		ForwardHandler:    forwardHandler,
 		DoForwardAlert:    (util.ForwardAllEvents || util.AllowType("alert")),
 		AlertPrefix:       alertPrefix,
+		BlacklistIOCs:     make([]string, 0),
```

Comment:
Leaving this at its default value, nil, should be ok. The only place where BlacklistIOCs is accessed seems to be a `for ... range` loop in `checkBlacklisted`.